api/generic-cti/index.rst

```diff
@@ -0,0 +1,45 @@
+Generic CTI
+***********
+
+This page describes the generic CTI API scopes and functionalities.
+
+.. include:: /api/generic-cti/cti-endpoint-limitations.include.rst
+
```

Comment:
If I want to find out if I want to implement this API with my BPX or if this API is the right one for me, I need to know in an introduction what I can do with this API. Technically the description if better but need to know the big picture/functional scope of the API.

This makes me miss the old introduction and feature list section here: https://docs.zammad.org/en/latest/cti/api-intro.html#introduction

Reply:
I personally don't like the mere introduction text with over 50% of a 2k screen about what's possible to be honest.

I also believe that we do give an idea on the situation:
![image](https://user-images.githubusercontent.com/6549061/156463303-365db52a-4b8a-41ca-9f6d-668b4e596847.png)


Also, if I'd wanted to know what a specific call is capable of it should stand with the affected endpoint and not three pages before the actual relevant part is mentioned.

Reply:
I have no more to add from my site.

Reply:
So I had a look at this topic again and this is what the condensed version looks like:
![image](https://user-images.githubusercontent.com/6549061/156565449-4ab7697b-6244-4ad1-95e3-4cf555bc2342.png)

vs:

![image](https://user-images.githubusercontent.com/6549061/156565475-94952afa-15d0-45be-8df7-216dcdb471d0.png)

Why is it so small? Because half of the feature list describes UI functions that belong to the user documentation (and are stated there: https://user-docs.zammad.org/en/latest/extras/caller-log.html). We're trying to reduce duplicate content as much as possible.

I also dropped this line:
`Support to allow an agent to set a DND - like state` - the reason for this is that I am unable to find such a function within Zammad. While Zammad seems to have such a DND like return it currently is unclear how it can be triggered.

If this is a real function, it is currently entirely missing from both, support knowledge and documentation.

Reply:
I like your proposal (condensed version) here too!